os/drivers/bluetooth/virtual_ble/Kconfig

```diff
@@ -0,0 +1,3 @@
+config VIRTUAL_BLE
+	bool "Virtual BLE Driver"
+	default n
```

Comment:
Can you add a newline? 
In my experience, it can occur a error in menuconfig

Reply:
Done